src/vertexcover/random_vertex_cover.jl

```diff
@@ -0,0 +1,98 @@
+"""
+    random_vertex_cover_it(g)
+
+### Implementation Notes
+Performs [Approximate Minimum Vertex Cover](https://en.wikipedia.org/wiki/Vertex_cover#Approximate_evaluation) once.
+Returns a vector of vertices representing the vertices in the Vertex Cover.
+### Performance
+O(|V|+|E|)
+### Approximation Factor
+2
+"""
+function random_vertex_cover_it(
+    g::AbstractGraph{T}
+    ) where T <: Integer 
+
+    nvg::T = nv(g)  
+    cover = Vector{T}()  
+    seen = zeros(Bool, nvg)
+    edge_list = Random.shuffle(collect(edges(g)))
```

Comment:
This is very expensive.

Reply:
The algorithms works like this:
```
while some_condition:
       Randomly choose an edge
       DO SOMETHING
end
```
I implemented it as:
```
edge_list = Random.shuffle(collect(edges(g)))
for e in edge_list
       some_condition || break
       DO SOMETHING
end
```